src/core/world/mod.rs

```diff
@@ -800,4 +789,22 @@ impl World {
     // self.debug_show_mem();
   }
 
+  fn find_movement_sounds(&mut self) -> Vec<(Pos, usize)> {
```

Comment:
I'm gonna rename this, since we wont just check for movement sounds exclusively.